ovirt_imageio/_internal/http.py

```diff
@@ -179,7 +179,9 @@ class Connection(http.server.BaseHTTPRequestHandler):
     # connections.  When the timeout expires we close the connection.
```

Comment:
Not sure how relevant is this, but I'd say you called me out on this once (?). There should be no space between the `Fixes` line and the sign-off line in the commit message.

But other than that, looks good.

Reply:
Yes, I fixed it locally, but I did not push since this must come after #72.